modules/nf-core/software/bwa/mem/main.nf

```diff
@@ -0,0 +1,46 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process BWA_MEM {
+    tag "$meta.id"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::bwa=0.7.17 bioconda::samtools=1.10" : null)
```

Comment:
Why not use samtoll 1.11? as in the other steps? Ah I guess, it is from the modules repo and not a local one, right?

Reply:
Because I did not check if there was a mulled container for it actually...
I just used the modules from nf-core/modules when there was one.
Was not planning to update tools in this PR.
I'm saving that for the future PRs